File: LeetCode/1758/golang/lc1758_test.go
package lc1758

import "testing"

func Test_minOperations(t *testing.T) {
	testcases := []struct {
		s        string
		expected int
	}{
		{"0100", 1},
		{"10", 0},
		{"1111", 2},
	}

	for i, tc := range testcases {
		actual := minOperations(tc.s)
		if actual != tc.expected {
			t.Errorf("Testcase minOperations#%02d (%v) failed: want %d, got %d",
				i+1, tc.s, tc.expected, actual)
		}
	}
}

File: LeetCode/1758/python/lc1758_test.py
import unittest

from lc1758 import Solution

class Test_minOperations(unittest.TestCase):
    def test(self):
        s = Solution()
        self.assertEqual(s.minOperations("0100"), 1)
        self.assertEqual(s.minOperations("10"), 0)
        self.assertEqual(s.minOperations("1111"), 2)

if __name__ == "__main__":
    unittest.main()
